### DISCUSSION TITLE
HMIP-BSL: Nach Firmware-Update funktionieren die LED-Taster nicht mehr

### DISCUSSION THREAD

**Discussion by DanK9000** [SukramJ/aiohomematic#1553] [General]
Hallo zusammen,

nach dem Firmware-Update vom HMIP-BSL auf Version 2.0.2 funktionieren die LED-Taster aufgrund eines fehlenden (neuen) Attributs nicht mehr.

![image](https://github.com/danielperna84/hahomematic/assets/165299294/7410dce8-fbb1-4e94-b040-50212916c7ca)

Ich musste in den CCU3 Programmen das Verhalten auf "Dauerhaft ein" stellen, damit die LEDs wieder in den entsprechenden Farben leuchten.
Wäre es möglich, dieses Attribut "Verhalten" in hahomematic einzubinden?

Gruß
Dan

**Comment by SukramJ** [comment]
Oder Du machst einfach ein Update.

**Reply by DanK9000** [reply]
Ich habe die Version 1.161.0.
Wo kann ich das Verhalten einstellen?
![image](https://github.com/danielperna84/hahomematic/assets/165299294/43e8e2ed-e729-4767-847d-a13d804e7003)

**Reply by SukramJ** [reply]
Effect

**Reply by DanK9000** [reply]
Perfekt.  Vielen Dank! 👍
